test/Microsoft.ML.OnnxTransformTest/OnnxTransformTests.cs

```diff
@@ -298,8 +298,16 @@ public void OnnxModelMultiInput()
                     {
                         getScoresa(ref buffera);
                         getScoresb(ref bufferb);
-                        Console.WriteLine(buffera.GetValues().ToArray());
+                        var suma = 0f;
+                        var sumb = 0f;
+                        foreach (var x in buffera.DenseValues())
```

Comment:
You can use `VectorUtils.Sum` here instead.

Reply:
Used linq to shorten.

---
In reply to: [236327775](https://github.com/dotnet/machinelearning/pull/1717#discussion_r236327775) [](ancestors = 236327775)